functional_tests/steps/bundles.py

```diff
@@ -53,3 +54,30 @@ def the_user_opens_page_chooser(context: Context) -> None:
 def the_locale_column_is_displayed(context: Context) -> None:
     modal = context.page.locator(".modal-body")
     modal.get_by_role("columnheader", name="Locale").is_visible()
+
+
+@step("the user can add Bundles created by user")
+def a_bundle_with_name_item_name_has_been_created_by_username(context: Context) -> None:
+    context.bundle = BundleFactory(created_by=context.user_data["user"])
+
+
+@step("Bundle has the creator removed")
+def created_by_has_been_deleted(context: Context) -> None:
+    context.bundle.created_by = None
+    context.bundle.save(update_fields=["created_by"])
```

Comment:
This sounds like it should be a `@given()` to be used at a beginning of a test, as a step should not directly manipulate any data in a functional test.